DEPLOYING.md

```diff
@@ -15,6 +15,11 @@ Examples:
   * Python: BP_FUNCTION=func.main. `func` is the name of the .py file. main is the `method`.
   * Java: BP_FUNCTION=function.Handler. `function` the package. `Handler` is the class that implements Function.
 
+Note:
```

Comment:
This is a nit, but "notes" generally look nice in a quote block

Reply:
Yes, good idea. Thanks!